spec/models/message_spec.rb

```diff
@@ -1,5 +1,78 @@
 require 'rails_helper'
 
 RSpec.describe Message, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  
+
+	context "When a message is created" do
+
+		before do
+			@message = Message.new
+	  	@message.id = 1
+			@message.created_at = "2016-10-22 14:34:00"
+			@message.updated_at = "2016-10-22 14:34:00"
+			@message.sender_id = "1"
+			@message.receiver_id = "2"
+			@message.message_body = "I want to talk to you!!!"
+			@message.appointment_date = "2016-10-23 16:00:00"
+  	end
+
+		it "is a new message" do
+			expect(@message).to be_a_new(Message)
+		end
+
+		it "should be a valid message object" do
+			expect(@message).to be_valid
+		end
+
+		it "is expected to save message properly" do
+			expect(@message.created_at).to eq("2016-10-22 14:34:00")
+			expect(@message.updated_at).to eq("2016-10-22 14:34:00")
+			expect(@message.sender_id).to eq("1")
+			expect(@message.receiver_id).to eq("2")
+			expect(@message.message_body).to eq("I want to talk to you!!!")
+			expect(@message.appointment_date).to eq("2016-10-23 16:00:00")
+		end
+
+	end
+
+	context "when message doesn't have receiver_id" do
+		before do
+			@message = Message.new
+	  	@message.id = 2
+			@message.created_at = "2016-10-22 14:25:02"
```

Comment:
@FottyM Yeah, just checked and it works fine without these. I thoght that as they were not allowed to be null in schema.rb they were mandatory. Should I remove these and create a new pull request? Allso should I use Time.now for appointment date too?